src/smart-components/group/groups.js

```diff
@@ -176,10 +179,24 @@ const Groups = () => {
         ]
       : []),
   ];
+
   const data = groups.map((group) =>
     group.platform_default || group.admin_default ? { ...group, principalCount: `All${group.admin_default ? ' org admins' : ''}` } : group
   );
-  const rows = createRows(isAdmin, data, selectedRows);
+
+  const fetchExpandTableRoleDetailsForGroup = (uuid) => dispatch(fetchRolesForGroup(uuid, defaultCompactSettings));
+  const fetchExpandTableMemberDetailsForGroup = (uuid) => dispatch(fetchMembersForGroup(uuid, defaultCompactSettings));
```

Comment:
Can we rename it to something shorter? Like...
```suggestion
  const fetchExpandedRoles = (uuid) => dispatch(fetchRolesForGroup(uuid, defaultCompactSettings));
  const fetchExpandedMembers = (uuid) => dispatch(fetchMembersForGroup(uuid, defaultCompactSettings));
```